sys-apps/exa/metadata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ya@sysdump.net</email>
+		<name>Georgy Yakovlev</name>
+		<description>Proxied maintainer. Please assign bugs to him.</description>
```

Comment:
Please don't introduce useless (obvious) descriptions. They only slow down bug assignment.

Reply:
@mgorny I've committed/squashed a change, removing redundant descriptions.
Also Updated referenced bug to #639098, since mine was a duplicate.